test/functional/lib/xdptests_km.vcxproj

```diff
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <ClCompile Include="tests.cpp" />
+  </ItemGroup>
+  <PropertyGroup>
+    <ProjectGuid>{c8280a34-02c5-4bde-9f4f-71efa06c8351}</ProjectGuid>
+    <TemplateGuid>{0a049372-4c4d-4ea0-a64e-dc6ad88ceca1}</TemplateGuid>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetName>xdptests_km</TargetName>
+    <UndockedType>drvlib</UndockedType>
+  </PropertyGroup>
+  <Import Project="$(SolutionDir)src\xdp.cpp.props" />
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <AdditionalIncludeDirectories>
+        $(SolutionDir)test\common\inc;
+        $(SolutionDir)test\functional\inc;
+        $(SolutionDir)submodules\wil\include;
+        $(FnPackagePath)include;
+        %(AdditionalIncludeDirectories)
+      </AdditionalIncludeDirectories>
+      <PreprocessorDefinitions>
+        NDIS683=1;
```

Comment:
why NDIS683? we already have a mix of NDIS685 and NDIS684 for reasons i don't understand, and now we're adding another version?

Reply:
Let's use 684 since that's what the product code uses